### PR TITLE
Categorization of utils

### PR DESCRIPTION
**Pull request by gurjit03**
* [x]  Passed [sanity tests](http://bit.ly/sttm-sanity-tests).
* [x]  Ran `npm test` & fixed newly introduced lint errors.
* [x]  Checked console for errors.

This PR adds categorization for the utils and ports them to typescript. Earlier we have all the utils in 1 single file, which requires lots of cognitive efforts to parse. Also, earlier implementation was getting hard to maintain

After categorization
image

<!-- New to markdown? Simply put an 'x' between [ ] to check it. Like [x] this. (No spaces).

### CODE REVIEW

File: src/js/util/visraam/get-visraam-class.ts [hunk 1]
```diff
@@ -0,0 +1,18 @@
+export const getVisraamClass = (verse: string, akharIndex: number, visraams: any) => {
```
[the remark this answers is not in the document]
Comment (gurjit03) [reply src/js/util/visraam/get-visraam-class.ts:1]: not sure @plug-n-play any hints how to see the type of visraams we have?

**Comment by saintsoldierx** [comment]
@gurjit03 ^ conflicts

File: src/js/util/visraam/get-visraam-class.ts (hunk 1 above)
[the remark this answers is not in the document]
Comment (saintsoldierx) [reply src/js/util/visraam/get-visraam-class.ts:1]: @ManjotS any input?

[the remark this answers is not in the document]
Comment (ManjotS) [reply src/js/util/visraam/get-visraam-class.ts:1]: visraam apparently shows up as string on int sometimes

**Pull request merged**